themes/default/content/docs/intro/concepts/state.md

```diff
@@ -118,7 +109,7 @@ To view your access tokens, or create a new one manually, view the <a href="http
 
 <img src="/images/docs/reference/state_tokens.png" alt="Pulumi.com Tokens Page" class="img-bordered">
 
-##### Logging Into a Self-Hosted Pulumi Service Backend
+### Self-Hosted Pulumi Service
```

Comment:
honestly i think this doesnt need its own heading, and the content could just be at the end of the previous section